src/main/java/core/basesyntax/services/WriteFileService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import core.basesyntax.dao.StoreDao;
+
+public interface WriteFileService {
+    void writeToFile(StoreDao dao, String fileName);

```

Comment:
Its will be better if all your methods will return some type (for instanse boolean). Its helps you while testing your code.